src/objectivec/oga_sequences.mm

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "error_utils.h"
+#import "oga_internal.h"
+#import "ort_genai_objc.h"
+
+@implementation OGASequences {
+  std::unique_ptr<OgaSequences> _sequences;
+}
+
+- (instancetype)initWithCXXPointer:(std::unique_ptr<OgaSequences>)ptr {
+  _sequences = std::move(ptr);
+  return self;
+}
+
+- (nullable instancetype)initWithError:(NSError**)error {
+  if ((self = [super init]) == nil) {
+    return nil;
+  }
+
+  try {
+    _sequences = OgaSequences::Create();
+    return self;
+  }
+  OGA_OBJC_API_IMPL_CATCH_RETURNING_NULLABLE(error)
+}
+
+- (size_t)count {
+  return _sequences->Count();
```

Comment:
nit: also catch exceptions here. even if it doesn't throw now, it's safer to treat all C++ code as potentially throwing.

Reply:
general: maybe use methods instead of properties so NSError may be used

Reply:
done